aldryn_config.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+
+from aldryn_client import forms
+
+
+class Form(forms.BaseForm):
+
+    META_SITE_PROTOCOL = forms.CharField('META_SITE_PROTOCOL', required=True)
+    META_SITE_DOMAIN = forms.CharField('META_SITE_DOMAIN', required=True)
+    META_SITE_TYPE = forms.CharField('META_SITE_TYPE', required=False)
+    META_SITE_NAME = forms.CharField('META_SITE_NAME', required=False)
+    META_INCLUDE_KEYWORDS = forms.CharField('META_INCLUDE_KEYWORDS', required=False)
+    META_DEFAULT_KEYWORDS = forms.CharField('META_DEFAULT_KEYWORDS', required=False)
+    META_IMAGE_URL = forms.CharField('META_IMAGE_URL', required=False)
+    META_USE_OG_PROPERTIES = forms.CheckboxField('META_USE_OG_PROPERTIES', required=False)
+    META_USE_TWITTER_PROPERTIES = forms.CheckboxField('META_USE_TWITTER_PROPERTIES', required=False)
+    META_USE_GOOGLEPLUS_PROPERTIES = forms.CheckboxField('META_USE_GOOGLEPLUS_PROPERTIESS', required=False)
+    META_USE_TITLE_TAG = forms.CheckboxField('META_USE_TITLE_TAG', required=False)
+    META_USE_SITES = forms.CheckboxField('META_USE_SITES', required=False)
```

Comment:
I think we can simplify the settings here:
- META_SITE_PROTOCOL is retrieavable from aldryn: I'd use a choice, in case
- META_SITE_DOMAIN: can we retrieve this from aldryn configuration in some way?
- META_SITE_NAME: can we retrieve this from aldryn configuration in some way?
- META_SITE_TYPE: Use a choice based on https://github.com/nephila/django-meta-mixin/blob/develop/meta_mixin/settings.py#L7
- META_INCLUDE_KEYWORDS force this to false (e.g.: remove from form and just set it to false in ``to_settings``)
- META_DEFAULT_KEYWORDS force this to false (e.g.: remove from form and just set it to false in ``to_settings``)
- META_USE_TITLE_TAG force this to false (e.g.: remove from form and just set it to false in ``to_settings``)